graphql/resolvers.ts

```diff
@@ -69,6 +71,22 @@ export default {
   Mutation: {
     changePw,
     createSubmission,
+    async acceptSubmission(_parent: void, args: ArgsUpdateSubmission) {
+      try {
+        if (!args) throw new Error('Invalid args')
+        return await updateSubmission({ ...args, status: 'passed' })
+      } catch (error) {
+        throw new Error(error)
+      }
+    },
+    async rejectSubmission(_parent: void, args: ArgsUpdateSubmission) {
+      try {
+        if (!args) throw new Error('Invalid args')
+        return await updateSubmission({ ...args, status: 'needMoreWork' })
+      } catch (error) {
+        throw new Error(error)
+      }
+    },
```

Comment:
Move to `submissionController`

Reply:
You should able to if you change the way this is written. Instead of calling a mutation from another mutation. Make a helper function to `updateSubmission`. That way you can just do a unit test for that function itself and then mock that function for the mutations.